ts/DataGrid/DataGrid.ts

```diff
@@ -185,6 +185,11 @@ class DataGrid {
      */
     public loadingWrapper?: HTMLElement;
 
+    /**
+     * Whether the loading indicator is shown.
+     */
+    private loadingShown = false;
+
```

Comment:
Is this parameter really needed? Maybe just check if `this.loadingWrapper` exists?